week2/exercise-template/readme.md

```diff
@@ -1,69 +1,28 @@
-<p align="center">
```

Comment:
Seems like this readme is from homework week 3?

Reply:
Ahh yes, good point!